data/data/agent/systemd/units/node-zero.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
 Description=Identify node zero to run OpenShift Assisted Installation Service on
 Wants=network-online.target
-After=network-online.target
+After=network-online.target agent-auth-token-status.service
+Requires=agent-auth-token-status.service
```

Comment:
This doesn't seem necessary?

Reply:
Yes, This PR was based on https://github.com/openshift/installer/pull/8802 and the changes as above are originating form that PR. Once https://github.com/openshift/installer/pull/8802 merges, I will rebase this PR. The actual changes meant for the current PR are under commit https://github.com/openshift/installer/pull/8804/commits/d92243506e9a154f9d7610a6c2fd194ba0d97404 and https://github.com/openshift/installer/pull/8804/commits/e6c62f878ea1271d03473623c54f08abe4a13210